src/Native/Windows/clrcompression/zlib-intel/README.txt

```diff
@@ -0,0 +1,11 @@
+Zlib-Intel is an optimized version of Zlib developed by Intel to take advantage of modern CPU features. 
```

Comment:
Just to confirm - the notices file already includes the appropriate ZLib reference due to the code on which this PR is based so we're good as is. Thanks for checking.